topology/graph/traversal/traversal.go

```diff
@@ -1668,6 +1677,18 @@ func (t *GraphTraversalValue) Error() error {
 	return t.error
 }
 
+func (t *GraphTraversalValue) assert() {
+	if t.error != nil {
+		return
+	}
+	if t.GraphTraversal == nil {
+		panic("GraphTraversalValue.GraphTraversal should not be nil!")
```

Comment:
I don't think we want to panic. In any case it will panic if using it set to null.

Reply:
@safchain having an explicit panic here will help identify the source of the issue ... if not then the panic further down the line will be much more difficult to debug ...

Reply:
It is clearly an error during coding phase, I would agree if the value will come from outside like from a user input but here I don't think we need helper to fix obvious coding mistakes.

Reply:
@hunchback 
please add these lines in your .git/hooks/pre-push

unset TEST_PATTERN
make
make fmt test
make test.functionals.compile

Reply:
@safchain - ok will remove assert()